test/test.js

```diff
@@ -8,4 +9,18 @@ describe('markdown-it-video', function requireMarkdownIt() {
     typography: true,
   }).use(require('../'));
   generate(path.join(__dirname, 'fixtures/video.txt'), md);
+
+  it('should not render two on a single line', function() {
```

Comment:
Sorry, you've done too much work here. I forgot this had a shortcut for describing these kinds of tests. Take this out and look instead at https://github.com/brianjgeiger/markdown-it-video/blob/develop/test/fixtures/video.txt for the best way to add these test cases.

Reply:
I did try this and it seemed to tokenize it strangely but I'll have another look at it.